docs/getting_started.rst

```diff
@@ -1,16 +1,119 @@
 Getting Started
 ===============
 
+Production setup
+----------------
+
+`Edx-enterprise` is developed as a pluggable application for the edx-platform and can't be used outside of edx-platform.
+
+`Edx-enterprise` is shipped with default edx-platform setup (see `edx-platform requirements file`_), so new installs
+should already have it set up and enabled. However, if you're running an older version (i.e. pre-Ficus) of edx-platform,
+manual setup must be performed. In `edxapp` virtualenv (`$current_release` is |release|)\ [#f1]_
```

Comment:
@bradenmacdonald Initially it read "if you're **upgrading** from older version", but than I thought it should be in ficus upgrade guidelines. It would be still usable for development setup (I don't think everyone just nukes their devstacks and rebuilds from scratch when new named release comes out), so I'll move it there.